website/content/docs/secrets/aws.mdx

```diff
@@ -332,6 +336,38 @@ secret_key     	HSs0DYYYYYY9W81DXtI0K7X84H+OVZXK5BXXXX
 security_token 	AQoDYXdzEEwasAKwQyZUtZaCjVNDiXXXXXXXXgUgBBVUUbSyujLjsw6jYzboOQ89vUVIehUw/9MreAifXFmfdbjTr3g6zc0me9M+dB95DyhetFItX5QThw0lEsVQWSiIeIotGmg7mjT1//e7CJc4LpxbW707loFX1TYD1ilNnblEsIBKGlRNXZ+QJdguY4VkzXxv2urxIH0Sl14xtqsRPboV7eYruSEZlAuP3FLmqFbmA0AFPCT37cLf/vUHinSbvw49C4c9WQLH7CeFPhDub7/rub/QU/lCjjJ43IqIRo9jYgcEvvdRkQSt70zO8moGCc7pFvmL7XGhISegQpEzudErTE/PdhjlGpAKGR3d5qKrHpPYK/k480wk1Ai/t1dTa/8/3jUYTUeIkaJpNBnupQt7qoaXXXXXXXXXX
 ```
 
+### STS Session Tokens
+
+~> **Notice:** Due to limitations in AWS, in order to use the `session_token`
+credential type, Vault **must** be configured with IAM user credentials. AWS
+does not allow temporary credentials (such as those from an IAM instance
+profile) to be used.
```

Comment:
```suggestion
<Important>
  AWS does not allow temporary credentials like those from an IAM instance
  profile. To use session tokens with Vault and AWS, you must configure Vault
  to use IAM user credentials.
</Important>
```

style correction: use new aside notation, mark the aside as important, and use active voice